Apache: refactor dashboards

mtojek opened the pull request:
<!-- Type of change
Please label this PR with one of the following labels, depending on the scope of your change:
- Bug
- Enhancement
- Breaking change
- Deprecation
-->

## What does this PR do?

<!-- Mandatory
Explain here the changes you made on the PR.
-->

This PR refactors dashboards for the Apache integration. Changes:
- [x] Add visualization for Connection stats: One for (apache.status.connections.async.writing, `apache.status.connections.async.keep_alive`, `apache.status.connections.async.closing`); one for (`apache.status.connections.total`)
- [x] Convert uptime information to human readable time.
- [x] Add a filter dropdown for `apache.status.hostname`
- [x] Add visualizations for `apache.status.requests_per_sec`, `apache.status.bytes_per_sec`.
- [x] Convert `apache.status.total_kbytes` to human readable representation depending on the volume of data, i.e. Mb, Gb, etc.
- [x] Convert total number of access requests to human readable representation (hundreds, thousands, millions, etc).


## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/CONTRIBUTING.md#tips-for-building-integrations) and this pull request is aligned with them.
- [x] I have verified that all datasets collect metrics or logs.


## How to test this PR locally

<!-- Recommended
Explain here how this PR will be tested by the reviewer: commands, dependencies, steps, etc.
-->

Use the howto guide to boot up Apache: https://github.com/elastic/elastic-package/blob/master/docs/howto/connect_service_with_elastic_stack.md
Verify if data appears on dashboards.

## Related issues

<!-- Recommended
Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Closes #123
- Relates #123
- Requires #123
- Supersedes #123
-->
- https://github.com/elastic/integrations/issues/370

## Screenshots

<!-- Optional
Add here screenshots presenting:
- Kibana UI forms presenting configuration options exposed by the integration
- dashboards with collected metrics or logs
-->

<img width="1671" alt="Zrzut ekranu 2020-12-9 o 17 27 31" src="https://user-images.githubusercontent.com/14044910/101657554-5305ac80-3a44-11eb-85c5-c174fcb47cf1.png">

![screencapture-localhost-5601-app-dashboards-2020-12-09-17_31_29](https://user-images.githubusercontent.com/14044910/101657653-6ca6f400-3a44-11eb-8afb-e555c7b23d9c.png)

elasticmachine commented:
Pinging @elastic/integrations (Team:Integrations)

elasticmachine commented:
## :green_heart: Build Succeeded


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-453/detail/PR-453/1//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-453/detail/PR-453/1//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-453/detail/PR-453/1//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://beats-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations%2FPR-453/detail/PR-453/1//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://integrations_453.docs-preview.app.elstc.co/diff)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats

* Build Cause: `Pull request #453 opened`

* Start Time: 2020-12-09T16:34:05.911+0000

* Duration: 20 min 51 sec




#### Test stats :test_tube:

| Test         | Results                         |
| ------------ | :-----------------------------: |
| Failed       | 0  |
| Passed       | 89  |
| Skipped      | 0 |
| Total        | 89   |


</p>
</details>

<!-- TEST RESULTS IF ANY-->


<!-- STEPS ERRORS IF ANY -->








<!--COMMENT_GENERATED_WITH_ID_comment.id-->

mtojek commented:
/test

sorantis commented:
Thanks for working on the dashboard @mtojek.
A few nits:
- Use margins between the panels
- Remove points from each line (i think it's done via point size input)

mtojek commented:
@sorantis 

> Use margins between the panels

Fixed

> Remove points from each line (i think it's done via point size input)

Fixed

Updated screenshots:

![screencapture-localhost-5601-app-dashboards-2020-12-10-13_57_02](https://user-images.githubusercontent.com/14044910/101776664-910cda00-3af1-11eb-88db-223b6f3c9ef6.png)

![screencapture-localhost-5601-app-dashboards-2020-12-10-13_57_15](https://user-images.githubusercontent.com/14044910/101776656-8d795300-3af1-11eb-84cf-65c32f604fb2.png)
